src/components/datetime.js

```diff
@@ -25,8 +25,8 @@ module.exports = function(app) {
 
             if(isNaN(dateVal.getDate())) {
               try {
-                const Moment = moment();
-                dateVal = new Date(eval($scope.component.defaultValue.toLowerCase().replace(/moment\(\)/i, "Moment")));
+                const moment = amMoment;
```

Comment:
```const``` is an ES6 item and we are not running this library through babel. We should not use it for this library.